### PR TITLE
Fix links due to repo rename

### PR DESCRIPTION
**Pull request by orizohar**
# Description

This PR does the following:
- Fix links that are broken by the rename of the repo from Samples to Quickstarts
- Removes numbering of quickstarts 

## Issue reference

https://github.com/dapr/quickstarts/issues/280

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] The sample code compiles correctly
* [X] You've tested new builds of the sample if you changed sample code
* [X] You've updated the sample's README if necessary

### CODE REVIEW

**Comment by pruthvidhodda** [comment]
Needs [changes in build.yaml](https://github.com/dapr/quickstarts/blob/master/.github/workflows/build.yml#L32-L47) for the build to pass

**Comment by orizohar** [comment]
> Needs [changes in build.yaml](https://github.com/dapr/quickstarts/blob/master/.github/workflows/build.yml#L32-L47) for the build to pass

Thanks! missed that one! just add those changes

**Comment by pruthvidhodda** [comment]
Needs changes in [this section](https://github.com/orizohar/samples/tree/quickstarts-rename/distributed-calculator#prerequisites-for-running-the-sample).

File: hello-kubernetes/README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Minikube users cannot see the external IP. Instead, you can use `minikube servic
 Here you can see that two ports are displayed. Both the ports have been injected when Dapr was enabled for this app. Additionally, in this sample the HTTP Port is used for further communication with the Dapr sidecar. 
 
 ## Step 5 - Deploy the Python App with the Dapr Sidecar
-Next, let's take a quick look at our python app. Navigate to the python app in the kubernetes sample: `cd samples/2.hello-kubernetes/python` and open `app.py`.
+Next, let's take a quick look at our python app. Navigate to the python app in the kubernetes sample: `cd samples/hello-kubernetes/python` and open `app.py`.
```
Comment (pruthvidhodda) [inline hello-kubernetes/README.md:99]: cd quickstarts/..............

File: hello-world/README.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ Follow [instructions](https://github.com/dapr/docs/blob/master/getting-started/e
 Now that we've locally set up Dapr, clone the repo, then navigate to the Hello World sample: 
 
 ```bash
-git clone [-b <dapr_version_tag>] https://github.com/dapr/samples.git
-cd samples/1.hello-world
+git clone [-b <dapr_version_tag>] https://github.com/dapr/quickstarts.git
+cd samples/hello-world
```
Comment (pruthvidhodda) [inline hello-world/README.md:28]: cd quickstarts/............

File: middleware/README.md [hunk 3]
```diff
@@ -25,10 +25,10 @@ helm install my-release stable/nginx-ingress
 
 1. Clone the sample repo, then navigate to the middleware sample:
 ```bash
-git clone [-b <dapr_version_tag>] https://github.com/dapr/samples.git
-cd samples/7.middleware/echoapp
+git clone [-b <dapr_version_tag>] https://github.com/dapr/quickstarts.git
+cd samples/middleware/echoapp
```
Comment (pruthvidhodda) [inline middleware/README.md:29]: cd quickstarts/....

**Comment by orizohar** [comment]
> Needs changes in [this section](https://github.com/orizohar/samples/tree/quickstarts-rename/distributed-calculator#prerequisites-for-running-the-sample).
> 
> Edit:
> 
> Here as well
> 
> [functions-and-keda](https://github.com/orizohar/samples/tree/quickstarts-rename/functions-and-keda#clone-the-sample-repository)
> [hello-dapr-slim](https://github.com/orizohar/samples/tree/quickstarts-rename/hello-dapr-slim#step-2---understand-the-code)

Both hello-dapr-slim and functions-and-keda will soon move to a new repo to be named Samples so this will be fixed by that

**Pull request merged**